test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -219,4 +189,26 @@ Describe "Invoke-Item tests on Windows" -Tags "CI","RequireAdminOnWindows" {
         Start-Process $testfilepath -Wait
         Test-Path $renamedtestfilepath | Should -BeTrue
     }
+
+    It "Should invoke an executable file without error" -Skip:(!$IsWindows) {
```

Comment:
Please clarify why we remove previous all-platform test and add this new for Windows only?

Reply:
The previous test expected UseShellExecute to only be available on Windows and the previous code was checking for different behavior on non-Windows.  Since I've changed it to only use ShellExecute on non-Windows if the target is not an executable, I should put back this test.